lms/djangoapps/mobile_api/social_facebook/groups/tests.py

```diff
@@ -0,0 +1,171 @@
+"""
+Tests for groups
+"""
+
+from django.core.urlresolvers import reverse
+from rest_framework.test import APITestCase
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from courseware.tests.factories import UserFactory
+
+
+class TestGroups(ModuleStoreTestCase, APITestCase):
+    """
+    Tests for /api/mobile/v0.5/groups/...
+    """
+    def setUp(self):
+        self.user = UserFactory.create()
+        self.client.login(username=self.user.username, password='test')
+
+    
+    '''
+        Group Creation and Deletion Tests
+    '''
+
+    def test_create_new_open_group(self):
+        # Create new group
+        url = reverse('create-delete-group', kwargs={'group_id': ''})
+        response = self.client.post(url, {'name': 'TheBestGroup',
+                                            'description': 'The group for the best people',
+                                            'privacy': 'open'})
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('id' in response.data)  # pylint: disable=E1103
+        # Delete the groupd just created
+        delete_group(self, response.data['id'])
+
+    def test_create_new_closed_group(self):
```

Comment:
Please use `ddt` for combining this test and the one above, that differs only by privacy setting.  You can see the documentation at http://ddt.readthedocs.org/en/latest/example.html.  And/or look for examples of ddt usage in the codebase.